### PR TITLE
fix: using yonius options typings

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-registry-api-js/pull/10#discussion_r575074145 |
| Dependencies | https://github.com/hivesolutions/yonius/pull/29 |
| Decisions | Using yonius `Options` type declarations. |
| Animated GIF | -- |

### CODE REVIEW

File: types/client.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Options } from "./options";
+import { Options } from "yonius";
 import { Project, ProjectCreate, ProjectPatch } from "./project";
```
Comment (gcandal) [inline types/client.d.ts:3]: Newline separating local from external imports

File: types/project.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { Client } from "./client";
-import { Options } from "./options";
+import { Options } from "yonius";
 import { Person } from "./person";
```
Comment (gcandal) [inline types/project.d.ts]: Reorder and separate imports

**Comment by gcandal** [comment]
Closing until the dependency is closed as well given that the review here is done and this will be ready to merge as soon as we settle on the naming for the type.

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
I'm not sure about this... Options should be something that inherits from the QueryParams and not the query params themselves, there's a reason why they are called options and not params like the underlying ones

**Comment by joamag** [comment]
@gcandal I'll discuss this with you

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**